---
id: 90ebf83e-9d8f-4141-bb66-25f3c04926f3
title: |
  mRNA-lipid nanoparticle COVID-19 vaccines: Structure and stability - ScienceDirect
author: |
  W. Abdelwahed, G. Degobert, S. Stainmesse, H. Fessi
date_saved: 2023-11-07 10:20:23
draft: true
---

# mRNA-lipid nanoparticle COVID-19 vaccines: Structure and stability - ScienceDirect
#Omnivore

[Read on Omnivore](https://omnivore.app/me/m-rna-lipid-nanoparticle-covid-19-vaccines-structure-and-stabili-18baa5ed454)

[Read Original](https://www.sciencedirect.com/science/article/pii/S0378517321003914)

date_saved: 2023-11-07 10:20:23


--- 

# Full Content: 

* [View **PDF**](https://www.sciencedirect.com/science/article/pii/S0378517321003914/pdfft?md5=735ae97fecb3791a1acfc1d96a960d56&pid=1-s2.0-S0378517321003914-main.pdf)

## Highlights

* •  
The current COVID-19 mRNA-LNPs vaccines must be stored at (ultra)low temperatures.
* •  
The core of LNPs contains mRNA, an ionizable cationic lipid, a neutral lipid and water.
* •  
mRNA hydrolysis is the determining factor for mRNA-LNP instability.
* •  
Improving stability through mRNA molecule optimization should be prioritized.
* •  
Other stabilization options: LNP optimization and dried formulation development.

## Abstract

A drawback of the current mRNA-lipid nanoparticle (LNP) COVID-19 vaccines is that they have to be stored at (ultra)low temperatures. Understanding the root cause of the instability of these vaccines may help to rationally improve mRNA-LNP product stability and thereby ease the temperature conditions for storage. In this review we discuss proposed structures of mRNA-LNPs, factors that impact mRNA-LNP stability and strategies to optimize mRNA-LNP product stability. Analysis of mRNA-LNP structures reveals that mRNA, the ionizable cationic lipid and water are present in the LNP core. The neutral helper lipids are mainly positioned in the outer, encapsulating, wall. mRNA hydrolysis is the determining factor for mRNA-LNP instability. It is currently unclear how water in the LNP core interacts with the mRNA and to what extent the degradation prone sites of mRNA are protected through a coat of ionizable cationic lipids. To improve the stability of mRNA-LNP vaccines, optimization of the mRNA nucleotide composition should be prioritized. Secondly, a better understanding of the milieu the mRNA is exposed to in the core of LNPs may help to rationalize adjustments to the LNP structure to preserve mRNA integrity. Moreover, drying techniques, such as lyophilization, are promising options still to be explored.

## 1\. Introduction

Of the many COVID-19 vaccines under development, the two vaccines that have shown the most promising results in preventing COVID-19 infection represent a new class of vaccine products: they are composed of messenger ribonucleic acid (mRNA) strands encapsulated in lipid nanoparticles (LNPs). The efficacy of these mRNA vaccines developed by BioNTech/Pfizer and Moderna is about 95% ([Baden et al., 2021](#b0020); [Polack et al., 2020](#b0340)) and they were the first mRNA vaccines to receive ‘emergency use authorization’ (by FDA) and ‘conditional approval’ by EMA. These mRNA COVID-19 vaccines encode the viral Spike (S) [glycoprotein](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/glycoprotein "Learn more about glycoprotein from ScienceDirect's AI-generated Topic Pages") of SARS-CoV-2 that includes two [proline](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/proline "Learn more about proline from ScienceDirect's AI-generated Topic Pages") substitutions (K986P and V987P mutations), in order to stabilize the prefusion conformation of the glycoprotein ([Wrapp et al., 2020](#b0465)). Upon intramuscular (IM) administration, the LNP system enables the uptake by host cells and the delivery of mRNA inside the cytosol, where the translation of the mRNA sequence into the [S protein](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/vitronectin "Learn more about S protein from ScienceDirect's AI-generated Topic Pages") occurs in the ribosomes. After post-translation processing by the host cells, the S protein is presented as a membrane-bound antigen in its prefusion conformation at the cellular surface, providing the antigen target for B cells. In addition, part of the temporally produced Spike proteins enter antigen presentation pathways, providing antigen recognition by T cells via MHC presentation of T-cell epitopes ([Verbeke et al., 2021](#b0430)). The EMA assessment report formulates the mechanism of action of mRNA vaccines at the injection site as follows: ‘Administration of LNP-formulated [RNA vaccines](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/rna-vaccine "Learn more about RNA vaccines from ScienceDirect's AI-generated Topic Pages") IM results in transient local inflammation that drives recruitment of neutrophils and antigen presenting cells (APCs) to the site of delivery. Recruited APCs are capable of LNP uptake and protein expression and can subsequently migrate to the local draining lymph nodes where T cell priming occurs ([EMA, 2020a](#b0085)).’ Because of this inherent innate immune activity, it is not necessary to formulate the mRNA vaccines with additional adjuvants. Interestingly, Pfizer/BioNTech and Moderna specifically use nucleoside-modified mRNA that decrease (rather than increase) the inherent mRNA [immunogenicity](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/immunogenicity "Learn more about immunogenicity from ScienceDirect's AI-generated Topic Pages"), underlining the need to properly balance the innate immune activity of mRNA vaccines (see below). The _in vivo_ antigen production post-administration that can be achieved with mRNA vaccines, together with the self-adjuvant properties of mRNA-LNP vaccines, ultimately leads to the efficient generation of [neutralizing antibody](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/neutralizing-antibody "Learn more about neutralizing antibody from ScienceDirect's AI-generated Topic Pages") responses and cellular immunity, decreasing the risk of developing COVID-19 for the vaccine recipients.

mRNA vaccines have several benefits over other types of vaccines. A general advantage of mRNA vaccines is that their development is relatively fast, as mRNA-LNPs are a true platform technology. After identification of the protective protein antigen(s) and sequencing the corresponding gene(s), the mRNA can be made within weeks ([Jackson et al., 2020](#b0160)). As the mRNAs encoding different antigens are chemically and physically highly similar, formulation design and manufacturing processes of new mRNA vaccines follow the same steps ([Petsch et al., 2012](#b0320)). Compared to replication deficient viral vectors, mRNA vaccines may be more efficacious for COVID-19 prevention. Unlike viral vector-based vaccines, they don’t generate immunity against the carrier. In this regard mRNA vaccines are similar to desoxyribonucleic acid (DNA)-based vaccines. DNA vaccines, however, still have a minute chance of potential genome integration. Moreover, in contrast to mRNA vaccines, DNA vaccines have shown rather low immunogenicity in early [clinical trials](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/clinical-trial "Learn more about clinical trials from ScienceDirect's AI-generated Topic Pages"), possibly because DNA-based vaccines need to gain access to the nucleus to exert their action, complicating efficient delivery. Overall, flexible design, standardized production processes and relatively short-lived cytoplasmic presence make mRNA vaccines very powerful, especially in a pandemic situation with rapidly mutating viruses.

However, one of the greatest challenges encountered when developing mRNA vaccines is their poor stability. Currently, most mRNA vaccines are administered IM, where the mRNA that is taken up by host cells leads to antigen expression ([Hassett et al., 2019](#b0140)). Early research on mRNA vaccines has demonstrated that naked mRNA is quickly degraded after administration ([Pardi et al., 2015](#b0300), [Wayment-Steele et al., 2020](#b0455)). Consequently, over the last few years efforts were made to improve the _in vivo_ stability of mRNA after administration. This led to ways to optimize the mRNA structure by slowing down its degradation (see under section ‘mRNA stability’). Another successful and currently widely used approach is to encapsulate and protect the mRNA in LNPs ([Pardi et al., 2015](#b0300)). This reduces premature mRNA degradation after administration and enhances delivery to the cytosol of antigen-presenting cells ([Liang et al., 2017](#b0250), [Lindsay et al., 2019](#b0255)).

Although progress has been made to enhance the stability _in vivo_ and efficacy of mRNA-LNP vaccines, much less attention has been paid to their stability during storage ([Crommelin et al., 2021](#b0060)). In order to effectively distribute a vaccine worldwide, it should have a sufficiently long shelf life, preferably at refrigerator temperatures (2–8 °C) or above. Currently, hardly any data is available in the public domain on what happens when mRNA-LNP formulations are stored for long periods of time. Moreover, it is unclear to what extent entrapping mRNA within LNPs influences the storage stability of the mRNA vaccine. Additionally, very little is known about the structure and morphology of LNPs formulated with mRNA, the chemical stability of the LNP components and the colloidal stability of the mRNA-LNP system. What is known now is that in order to store the current mRNA COVID-19 vaccines for longer periods of time, they have to be frozen. The current mRNA COVID-19 vaccines of Moderna and BioNTech/Pfizer have to be kept between −15 and −25 °C and between −60 and −90 °C, respectively ([EMA, 2020a](#b0085), [EMA, 2021](#b0080)). To date, the degradation processes and the reasons why storage temperature requirements differ, are not fully understood.

The requirement of storing the mRNA-LNPs in a frozen state hampers vaccine distribution. Especially, the very low temperature of −60 to −90 °C is a major obstacle when it comes to vaccine transport, storage and distribution among end-users worldwide. Most other vaccines can be stored at 2–8 °C. Clearly, there is a need and opportunity to find ways of stabilizing mRNA-LNP vaccines to allow non-frozen storage. This review provides an overview of approaches to make mRNA vaccines more stable, so that they can be stored longer at less extreme temperatures. To explore the topic, the characteristics of mRNA-LNP vaccines and their influence on storage stability are discussed. This information is used to identify the reasons for mRNA vaccine instability and to explore technological options for stability improvement.

## 2\. Overview of mRNA vaccines

The composition of mRNA-LNP vaccines is fundamental to their stability. In the development of vaccines against SARS-CoV-2, a variety of different mRNA vaccine candidates have been created. Currently, there are 10 different mRNA COVID-19 vaccines that have progressed to [clinical trials](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/clinical-trial "Learn more about clinical trials from ScienceDirect's AI-generated Topic Pages") ([World Health Organization, 2021](#b0460)). The SARS-CoV-2 mRNA vaccines either use conventional mRNA or self-amplifying mRNA (SAM). There are currently three ‘conventional’ mRNA vaccines in use or in advanced clinical trials that encode the full [S protein](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/vitronectin "Learn more about S protein from ScienceDirect's AI-generated Topic Pages"). These are the mRNA-1273 vaccine by Moderna, BNT162b2/Comirnaty by BioNTech/Pfizer and CVnCoV by CureVac ([Table 1](#t0005)). A detailed comparison of these three mRNA COVID-19 vaccines including their differences and similarities in mRNA structure and LNP design has been provided in several other reviews ([Kim et al., 2021](#b0210); [Verbeke et al., 2021](#b0430)). The following sections aim to give an overview of the function and characteristics of the mRNA component and the LNP delivery system in these vaccines, as they play a critical role in the stability of mRNA vaccines, both upon _in vivo_ administration and during storage.

Table 1. Information about the three mRNA-LNP drug products that are presently used or in clinical phase III trials. For comparison reasons, drug product information for Onpattro (an siRNA-LNP drug product) has been added.

| **Category**                                                                                    | **siRNA**                                                                                                                                                                                                                                                                | **Pfizer-BioNTech mRNA vaccine**                                                                                                                                                                                                                    | **Moderna mRNA vaccine**                                                                                                                                                                                                                                                                | **Curevac mRNA vaccine candidate**                                              |
| ----------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------- |
| **Name product**                                                                                | Onpattro \*patisiran                                                                                                                                                                                                                                                     | BNT162b2; Comirnaty                                                                                                                                                                                                                                 | mRNA-1273                                                                                                                                                                                                                                                                               | CVnCoV                                                                          |
| mRNA dose; route of administration                                                              | 0.3 mg/kg, intravenous                                                                                                                                                                                                                                                   | 30 µg; intramuscular                                                                                                                                                                                                                                | 100 µg; intramuscular                                                                                                                                                                                                                                                                   | 12 µg; intramuscular                                                            |
| Lipid nanoparticle components                                                                   | DLin-MC3-DMA: (6Z,9Z,28Z,31Z)-heptatriaconta-6,9,28,31-tetraen-19-yl-4-(dimethylamino)butanoate1,2-Distearoyl-_sn_\-glycero-3-phosphocholine (DSPC)PEG2000-DMG = Alpha-(3′-{\[1,2-di(myristyloxy)propanoxy\] carbonylamino}propyl)-ω-methoxy, polyoxyethyleneCholesterol | 0.43 mg ALC-0315 = (4-hydroxybutyl) azanediyl)bis (hexane-6,1-diyl)bis(2-hexyldecanoate)0.05 mg ALC-0159 = 2-\[(polyethylene glycol)-2000\]-N,N ditetradecylacetamide0.09 mg 1,2-Distearoyl-_sn_\-glycero-3-phosphocholine (DSPC)0.2 mg Cholesterol | SM-102 (heptadecan-9-yl 8-((2-hydroxyethyl) (6-oxo-6-(undecyloxy) hexyl) amino) octanoate}PEG2000-DMG = 1-monomethoxypolyethyleneglycol-2,3-dimyristylglycerol with polyethylene glycol of average molecular weight 20001,2-Distearoyl-_sn_\-glycero-3 phosphocholine (DSPC)Cholesterol | Cationic lipid (Acuitas Therapeutics)PhospholipidCholesterolPEG-lipid conjugate |
| Molar lipid ratios (%)ionizable cationic lipid : neutral lipid : cholesterol : PEG-ylated lipid | 50:10:38.5:1.5                                                                                                                                                                                                                                                           | 46.3:9.4:42.7:1.6                                                                                                                                                                                                                                   | 50:10:38.5:1.5                                                                                                                                                                                                                                                                          | 50:10:38.5:1.5                                                                  |
| Molar N/P ratios[a](#tblfn1)                                                                    | 3                                                                                                                                                                                                                                                                        | 6                                                                                                                                                                                                                                                   | 6[b](#tblfn2)                                                                                                                                                                                                                                                                           | 6[b](#tblfn2)                                                                   |
| Buffer                                                                                          | Potassium phosphate, monobasic, anhydrousSodium phosphate, dibasic, heptahydratepH \~ 7                                                                                                                                                                                  | 0.01 mg Potassium dihydrogen phosphate0.07 mg Disodium hydrogen phosphate dihydrate pH 7–8                                                                                                                                                          | Tris (tromethamine)pH 7–8                                                                                                                                                                                                                                                               | ?pH                                                                             |
| Other excipients                                                                                | Sodium chlorideWater for injection                                                                                                                                                                                                                                       | 0.01 mg Potassium chloride0.36 mg Sodium chloride6 mg SucroseWater for injection                                                                                                                                                                    | Sodium acetateSucroseWater for injection                                                                                                                                                                                                                                                | Saline                                                                          |

\*NDA 210922 ONPATTRO (patisiran) Lipid Complex Injection; Addendum to Drug Product Quality Review ([FDA, 2017](#b0130)).

a

N = ionizable cationic lipid (nitrogen), P = nucleotide (phosphate).

b

Estimate.

### 2.1\. mRNA engineering for optimum _in vivo_ stability and translation capacity

Because of the negative charges on its phosphate groups, mRNA is a polyanionic macromolecule in pH ranges typically used for parenterals ([Lipfert et al., 2014](#b0260)). A first obstacle for mRNA vaccines is that naked mRNA is quickly degraded upon injection by [ribonucleases](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/ribonuclease "Learn more about ribonucleases from ScienceDirect's AI-generated Topic Pages") (RNase), which are abundant in the extracellular environment. Second, the internalization of mRNA inside the cell is detected by intracellular RNA sensors, including endosomal Toll-like receptors (TLR) and cytoplasmic [nucleic acid](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/nucleic-acid "Learn more about nucleic acid from ScienceDirect's AI-generated Topic Pages") sensors. Binding of mRNA to these host defense receptors activates innate immune pathways, leading to the expression of hundreds of genes. One the one hand, this may provide an adjuvant effect on the vaccine potency. On the other hand, it establishes an antiviral state in cells, which strongly reduces the mRNA intracellular stability and translation ([Pepini et al., 2017](#b0315)). Following internalization, mRNA strands need to be recruited into the ribosomes to enable the expression of the encoded protein. The protein synthesis rate and the functional half-life of mRNA can be drastically increased through mRNA engineering. The typical elements of an mRNA strand for inclusion in an mRNA vaccine are schematically presented in [Fig. 1](#f0005).

![](https://proxy-prod.omnivore-image-cache.app/0x379,s7jH-4PMECcVMacY4kyJahXNZqEHXGycYo5N_6F6gVpQ/https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr1.jpg)
1. [Download : Download high-res image (251KB)](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr1%5Flrg.jpg "Download high-res image (251KB)")
2. [Download : Download full-size image](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr1.jpg "Download full-size image")

Fig. 1. Structural elements of _in vitro_ transcribed (IVT) mRNA. Each of these elements can be optimized and modified in order to modulate the stability, translation capacity, and immune-stimulatory profile of mRNA. Courtesy of [Verbeke et al. (2019](#b0425)).

Many efforts have been made to increase the _in vivo_ stability and translation capacity of the mRNA molecule, while avoiding unwanted innate immune activation. One prevalent idea is that this can be achieved by optimizing the regulatory regions of the mRNA: the 5′ cap, poly-A tail and untranslated regions (UTRs). The UTRs are parts of the mRNA that flank the coding region of the mRNA and regulate its stability and translation. The poly-A tail also regulates stability, as its shortening and eventual removal leads to mRNA degradation. The 5′ cap structure is important for protein production and the recruitment of translation initiation factors ([Pardi et al., 2018](#b0295)). Furthermore, mRNA with maximized GC (guanine-cytosine) content in combination with codon optimization, i.e., selection of ‘frequent codons’ in the coding region, leads to enhanced stability and translation ([Thess et al., 2015](#b0415)). Another critical determinant is the secondary structure of mRNA, which can be stabilized by changing the primary sequence through codon optimizations and computational tools. Building secondary structures in the mRNA –except in the 5′ UTR region– by choosing ‘highly structured coding sequences’ leads to higher translation _in vivo_ as well, because of a longer functional half-life ([Mauger et al., 2019](#b0265)). Alternatively, Mauger et al. demonstrated that the incorporation of naturally occurring modified [uridines](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/uridine "Learn more about uridines from ScienceDirect's AI-generated Topic Pages"), such as the use of 1-methyl-pseudouridine (m1Ψ) instead of uridine, induces global changes in the secondary structure of mRNA which correlated with high protein expression. Importantly, the introduction of these modified uridines inside the mRNA construct is currently the most efficient way to minimize the cellular recognition of mRNA by RNA-binding proteins that are involved in the innate immune reaction to foreign mRNA. This, in turn, enhances biological stability and translational capacity, while reducing the reactogenicity of mRNA vaccines. Moreover, there are also indications that m1Ψ increases base stacking and the melting point of mRNA, thereby making mRNA more stable ([Mauger et al., 2019](#b0265), [Zhao and He, 2015](#b0485)). This could mean that the incorporation of 1mΨ, as is done for the COVID-19 vaccines from Moderna and BioNTech/Pfizer, also affects the stability of mRNA before administration. CureVac followed a different path, as outlined by Thess et al. ([Thess et al., 2015](#b0415)). An interesting analysis of the choices CureVac made in engineering the CVnCoV mRNA was recently published ([Hubert, 2021](#b0150)). However, even when these structural elements are optimized, [IM injection](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/intramuscular-injection "Learn more about IM injection from ScienceDirect's AI-generated Topic Pages") of naked mRNA through a syringe does not lead to a robust immune response, probably because of the poor cell transfection capacity of naked mRNA and the susceptibility to RNase ([Pardi et al., 2015](#b0300)). Importantly, this illustrates that optimization of only the mRNA structure is not sufficient for creating effective mRNA vaccines: additional protection and delivery systems are needed ([Kauffman et al., 2016](#b0195), [Sahin et al., 2014](#b0370)).

The other type of mRNA vaccines, SAMs, do not only encode the target antigen but also [RNA polymerase](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/rna-polymerase "Learn more about RNA polymerase from ScienceDirect's AI-generated Topic Pages") encoding ‘self-amplifying’ factors derived from [Alphavirus](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/alphavirus "Learn more about Alphavirus from ScienceDirect's AI-generated Topic Pages"). Typically, they consist of 9 kb mRNA nucleotides compared to 2–4 kb for non-replicating mRNA vaccines. The [SAM vaccine](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/mrna-vaccine "Learn more about SAM vaccine from ScienceDirect's AI-generated Topic Pages") candidates were developed with the intent of forgoing the typical ‘prime-boost’ regimen of a two-dose strategy and instead focusing on a single injection per recipient. Because of their replication competence, the injected dose for SAM vaccines is lower than for conventional mRNA vaccines and one dose might be sufficient for protection ([Erasmus et al., 2020](#b0100), [Vogel et al., 2018](#b0440)). When SAMs are translated in the host cell, an [RNA replicase](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/rna-directed-rna-polymerase "Learn more about RNA replicase from ScienceDirect's AI-generated Topic Pages") synthesizes negative-sense RNA intermediates complementary to the coding mRNA template ([Fig. 1](#f0005)). These are in turn transcribed to many coding mRNA molecules, leading to prolonged and enhanced antigen expression ([Bloom et al., 2020](#b0035)). Both SAM vaccines encode the full S protein and the highest dose for these vaccines in clinical trials was more than tenfold lower than the typical doses used for the conventional mRNA vaccines ([Ward and McCormack, 2021](#b0450)). The two SAM vaccines in clinical development are nCoVsaRNA by the Imperial College London and ARCT-021 by Arcturus/Duke-NUS (both as mRNA-LNP).

### 2.2\. LNPs as delivery system

To overcome these transfection problems with naked mRNA, protecting delivery systems have been developed. Currently, the leading mRNA COVID-19 vaccines ([Table 1](#t0005)) are all utilizing LNP technology. This illustrates the successes achieved with this type of [nanoparticle](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/nanoparticle "Learn more about nanoparticle from ScienceDirect's AI-generated Topic Pages") to stabilize mRNA and successfully deliver it into cells ([Pardi et al., 2015](#b0300)). The LNPs in mRNA COVID-19 vaccines consist of four main components (cf. [Table 1](#t0005)): a neutral [phospholipid](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/phospholipid "Learn more about phospholipid from ScienceDirect's AI-generated Topic Pages"), cholesterol, a polyethylene-glycol (PEG)-lipid, and an ionizable cationic lipid. The latter contains positively charged ionizable amine groups (at low pH) to interact with the anionic mRNA during particle formation and also facilitate membrane fusion during internalization ([Evers et al., 2018](#b0110), [Reichmuth et al., 2016](#b0350)). In addition, PEG-lipid is used to control the particle size and act as a steric barrier to prevent aggregation during storage. Together with the mRNA, these components form particles of about 60–100 nm in size by using a rapid mixing production technique ([Evers et al., 2018](#b0110)). The SARS-CoV-2 vaccine candidates nCoVsaRNA and ARCoV, for example, have average particle sizes of 75 nm and 89 nm, respectively ([McKay et al., 2020](#b0270), [Zhang et al., 2020a](#b0470)).

A key aspect of LNPs and the characteristic that makes them different from [liposomes](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/liposome "Learn more about liposomes from ScienceDirect's AI-generated Topic Pages") (spherical vesicles with at least one lipid bilayer and an aqueous core) is the presence of lipids in the core, although data from several studies indicate that water is also present to some extent ([Yanez Arteta et al., 2018](#b0010), [Kulkarni et al., 2018](#b0225), [Brader et al., 2021](#b0040)). This would mean that the mRNA could be exposed to an aqueous environment, even when it is encapsulated. This type of core structure has been previously found in both unloaded and siRNA (small interfering RNA) containing LNPs, as demonstrated by cryogenic [transmission electron microscopy](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/transmission-electron-microscopy "Learn more about transmission electron microscopy from ScienceDirect's AI-generated Topic Pages") (cryo-TEM) ([Kulkarni et al., 2019](#b0230), [Kulkarni et al., 2018](#b0225)). Similarly, studies of mRNA-LNPs have shown electron dense cores ([Fig. 2](#f0010)) ([Yanez Arteta et al., 2018](#b0010), [Eygeris et al., 2020](#b0115), [Leung et al., 2015](#b0240), [Patel et al., 2020](#b0310), [Tanaka et al., 2020](#b0410)). [Thionine](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/thionine "Learn more about Thionine from ScienceDirect's AI-generated Topic Pages") was used for cryo-TEM contrast enhancement ([Brader et al., 2021](#b0040)).

![](https://proxy-prod.omnivore-image-cache.app/0x372,sq20f4Tiy1IRVHIhrhpdUEBe5TLE2Ph3A1OJhnhNh-RE/https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr2.jpg)
1. [Download : Download high-res image (820KB)](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr2%5Flrg.jpg "Download high-res image (820KB)")
2. [Download : Download full-size image](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr2.jpg "Download full-size image")

Fig. 2. Cryo-TEM image of mRNA-LNP showing ‘bleb’ structures with distinctly different electron density. Adapted from [Brader et al. (2021](#b0040)) with permission.

Although their lipid core is a common feature of mRNA-LNPs, characteristics of the exact structure of this core and its dependence on lipid ingredients (molar ratios) and the localization of the mRNA are still under debate (see [Fig. 3](#f0015)). mRNA is certainly located inside the LNPs, as determined by the RiboGreen assay. RiboGreen is a dye that exhibits fluorescence when bound to single stranded mRNA, but cannot enter the LNPs. In mRNA-LNP formulations, such as those used in mRNA vaccines, the fraction of accessible mRNA is very low ([Yanez Arteta et al., 2018](#b0010), [Patel et al., 2020](#b0310)) and thus, encapsulation efficiencies, derived from RiboGreen assays, are typically > 90%. Taken together, the cryo-TEM ([Fig. 2](#f0010)) and encapsulation evidence shows that the mRNA-LNPs form nanoparticles with encapsulated mRNA that is protected from the external medium. There are 3 models proposed for the structure of mRNA encapsulated by LNPs; these mainly originate from siRNA-LNPs analysis ([Fig. 3](#f0015)).

![](https://proxy-prod.omnivore-image-cache.app/0x258,shGPTS3JenrQkMHZO_XsmaeojVePxOU1qJYHfq5letrc/https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr3.jpg)
1. [Download : Download high-res image (768KB)](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr3%5Flrg.jpg "Download high-res image (768KB)")
2. [Download : Download full-size image](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr3.jpg "Download full-size image")

Fig. 3. Schematic representation of the proposed models for siRNA-LNP and mRNA-LNP structure. A: multilamellar vesicles; B: [nanostructure](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/nanomaterial "Learn more about nanostructure from ScienceDirect's AI-generated Topic Pages") core; C: homogeneous core shell as discussed by [Viger-Gravel et al. (2018](#b0435)). Courtesy of the authors.

The dawn of mRNA vaccA limitation of this is that the encapsulated cargo affects the structure of LNPs: siRNA is very different from mRNA in structure and size ([Table 2](#t0010)) and the molar N/P (ionizable cationic lipid over phosphate) molar ratios differ, 3 versus 6, respectively ([Table 1](#t0005)). mRNA is at least 100-fold larger than siRNA and this affects the structure of the LNPs. Besides, there are indications that mRNA is located in the core of the LNPs and siRNA more towards the surface and mRNA can form ‘blebs’ ([Fig. 2](#f0010)) ([Viger-Gravel et al., 2018](#b0435)). The composition of the bleb part of the LNP is a matter of debate ([Leung et al., 2015](#b0240), [Brader et al., 2021](#b0040)). The latter state: “mRNA can dissociate from the charged lipid to reside within a solvent-filled bleb compartment.”

Table 2. Differences between mRNA and siRNA molecules.

| Empty Cell               | mRNA            | siRNA                 |
| ------------------------ | --------------- | --------------------- |
| Molecular weight (g/mol) | ≥ 106           | 104                   |
| Molecular conformation   | Single stranded | Double stranded       |
| 5′ end                   | 5′ cap          | Phosphorylated 5′ end |
| 3′ end                   | Poly-A tail     | Hydroxylated 3′ end   |

The multilamellar vesicle model for mRNA-LNPs ([Fig. 3](#f0015)A) is unlikely to be correct. It does not correspond to the electron dense cores found by TEM of the mRNA-LNPs. Currently, most researchers hold the view that mRNA-LNPs are best described by the core–shell model ([Fig. 3](#f0015)B & C). This means that the nanoparticles have a surface layer and an amorphous, isotropic core. Viger-Gravel _et al._ used NMR spectroscopy to elucidate an LNP structure and they make the case that two types of core are possible ([Viger-Gravel et al., 2018](#b0435)). They describe the model of an amorphous core containing water pores surrounded by inverted cationic lipids ([Fig. 3](#f0015)B). They also postulate that the lipids in the core could be homogeneously dispersed with small water pockets in between ([Fig. 3](#f0015)C). The latter corresponds more to theexperimental results for their siRNA-LNPs and mRNA-LNPs.

Arteta _et al._ based their mRNA-LNP structure-model on cryo-TEM, small-angle X-ray scattering (SAXS) and small-angle neutron scattering (SANS) measurements. They found that 1,2-distearoyl-sn-glycero-3-phosphocholine (DSPC) and PEG-lipid as well as part of the ionizable cationic lipid and cholesterol are located on the surface of the LNPs. Inside the core, the main part of the ionizable cationic lipid, cholesterol (dependent on its concentration), water and mRNA are present. Interestingly, their research indicates that the isotropic LNP core consists for 24% (volume fraction) of water. They propose that mRNA is located inside water cylinders, which are surrounded by cationic lipids ([Fig. 4](#f0020)). This would mean that the mRNA is –at least partly– exposed to water inside the LNPs, which likely contributes to its instability upon storage under non-frozen conditions ([Arteta et al., 2018](#b0010)). Comparable results were reported by [Sebastiani et al. (2021](#b0385)).

![](https://proxy-prod.omnivore-image-cache.app/0x330,sUeIcLrrgqlNM2mSZPYNFN-y3GyOaaPJwbyNbnHR5LZo/https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr4.jpg)
1. [Download : Download high-res image (237KB)](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr4%5Flrg.jpg "Download high-res image (237KB)")
2. [Download : Download full-size image](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr4.jpg "Download full-size image")

Fig. 4. Schematic representation of the mRNA-water cylinders in the core of mRNA- LNPs ([Arteta et al., 2018](#b0010)). Courtesy of the authors.

It would, therefore, be interesting to study how mRNA interacts with water and ionizable cationic lipids in the LNP. mRNA is hydrophilic; it can interact electrostatically and through [hydrogen bonds](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/hydrogen-bond "Learn more about hydrogen bonds from ScienceDirect's AI-generated Topic Pages") with ionizable cationic lipids (apparent pKa < 6.5) ([Buschmann et al., 2021](#b0055)). This depends on the pH inside the LNP. If the LNP shells are permeable to protons –which is likely, as ionized dyes such as 2-(p-toluidino)-6-napthalene [sulfonic acid](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/sulfonic-acid "Learn more about sulfonic acid from ScienceDirect's AI-generated Topic Pages") (TNS) and thionine can enter the LNP core ([Jayaraman et al., 2012](#b0170), [Brader et al., 2021](#b0040)) – the pH would be similar to the rest of the formulation, around 7 to 8, meaning that most of the ionizable cationic lipids would be uncharged. However, as the ionizable cationic lipids are stacked in the core, they may show [polyelectrolyte](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/polyelectrolyte "Learn more about polyelectrolyte from ScienceDirect's AI-generated Topic Pages") behavior, leading to deviations of the Henderson-Hasselbalch equation, i.e., ‘smearing out’ of the titration curve ([Pierrat and Lebeau, 2015](#b0325)). Moreover, interaction between the mRNA and the ionizable cationic lipids might affect the ionization behavior. For siRNA only weak electrostatic interactions were found with the ionizable cationic lipid, indicating that at least for siRNA-LNP formulations the pH inside approaches or equals the pH outside ([Viger-Gravel et al., 2018](#b0435)). For mRNA-LNPs, no such experimental studies have been performed yet. A molecular dynamics simulation study of the complexation of mRNA and a cationic lipid demonstrated lipid-lipid cluster formation and lipid-mRNA cluster formation. Both electrostatic and hydrogen bonds were driving the cationic lipid and mRNA interactions ([Rissanou et al., 2020](#b0355)).

Another interesting finding by Arteta et al. is that the shell of their mRNA-LNPs is a monolayer ([Yanez Arteta et al., 2018](#b0010), [Verbeke et al., 2019](#b0425)). Other researchers propose that the outer shell of mRNA-LNPs consists of one or multiple bilayers, based on cryo-TEM analysis ([Fig. 2](#f0010)) ([Patel et al., 2020](#b0310), [Tanaka et al., 2020](#b0410)) or SANS ([Sebastiani et al., 2021](#b0385)). These findings indicate that assessment of the nature of the mRNA-LNP shell with these techniques is difficult and/or that multiple types of mRNA-LNP structures may exist, dependent on, [e.g.](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/eudragit "Learn more about e.g. from ScienceDirect's AI-generated Topic Pages"), the nature of the lipids and the preparation protocol for the mRNA-LNPs. This, in turn, could have implications for the stability of different formulations. In conclusion, questions remain around the structure of mRNA-LNPs and the interactions between the encapsulated mRNA and the various lipid components.

Analysis of the components of the various mRNA-COVID-19 vaccines has shown that there are common features, but also significant differences ([Table 1](#t0005)). The LNP formulation, the use of modified-nucleosides, the GC content and the differences in length between the conventional mRNA and SAM vaccines might influence the physical and chemical stability of these mRNA-COVID-19 vaccines during storage.

## 3\. In vitro stability of mRNA vaccines

As mentioned in the introduction, one of the main obstacles for the distribution of the currently approved mRNA-COVID-19 vaccines is that they have to be stored in frozen form (cf. [Crommelin et al., 2021](#b0060)). At refrigerator temperatures, 2–8 °C, the Pfizer/BioNTech and Moderna vaccines are stable for 5 and 30 days, respectively. Both companies provide detailed handling instructions for the end-user ([EMA, 2020b](#b0090), [EMA, 2021](#b0080)). Interestingly, the vaccine candidate by CureVac is reported to be stable for 3 months at refrigerator temperatures and at −60 °C ([CureVac, 2020](#b0065)). These are the only manufacturers who have released their long-term storage conditions at present. Such temperature requirements severely impact the logistics of the storage, transport, and distribution of these vaccines. However, to date little information on optimization of the stability of mRNA vaccines can be found in the public domain. This section aims to give an overview of the factors that influence the stability of the components of mRNA-LNP vaccines and discuss the methods to analyze this stability.

### 3.1\. mRNA stability

One factor that strongly influences the required storage conditions is the stability of the mRNA. As discussed above under Section 2.1, the structure of the mRNA molecule is specifically designed to increase the translation of the target antigen _in vivo_. A special feature of mRNA is that even one change (strand break, or oxidation of the bases) in the long mRNA strand (typically between 1000 and 5000 nucleotides long) can stop translation ([Klauer and van Hoof, 2012](#b0215)). This makes mRNA vaccines quite different from other vaccines in which small changes of the antigens do not necessarily have a measurable effect on their efficacy. Consequently, for mRNA vaccines, it is critical to monitor the integrity of the full molecule.

There are several ways in which mRNA degradation can occur; one can discern chemical and physical degradation. Chemical degradation encompasses the modifications of bonds in the mRNA molecule. Physical instability includes denaturation (loss of secondary and tertiary structure), which has different –likely less significant– consequences for the activity of mRNA than denaturation has for the activity of protein biologics. However, denaturation also comprises processes such as aggregation and precipitation, which negatively affect mRNA translation. In a review on the stability of [nucleic acids](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/nucleic-acid "Learn more about nucleic acids from ScienceDirect's AI-generated Topic Pages") Pogocki and Schöneich state that chemical degradation plays a larger role in the degradation of small nucleic acids than physical instability and that is probably even more true for large structures such as mRNA ([Pogocki and Schöneich, 2000](#b0335)).

Chemical degradation of mRNA _in vitro_ mainly occurs through hydrolysis and oxidation. Hydrolysis predominantly occurs via the [phosphodiester bonds](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/phosphodiester-bond "Learn more about phosphodiester bonds from ScienceDirect's AI-generated Topic Pages") that form the backbone of the mRNA molecule ([Fig. 5](#f0025)). The 2′ OH group on the [ribose](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/ribose "Learn more about ribose from ScienceDirect's AI-generated Topic Pages") plays a crucial role as the transesterification reaction leading to a mRNA strand break starts by a nucleophilic attack by the 2′OH group on the [phosphate ester](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/organophosphate "Learn more about phosphate ester from ScienceDirect's AI-generated Topic Pages") bond leading to a break at the P-O5′ ester bond ([Fig. 5](#f0025)). This process requires water and can be catalyzed by [nucleases](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/nuclease "Learn more about nucleases from ScienceDirect's AI-generated Topic Pages"), but also by the mRNA molecule itself and other exogenous factors like Brønsted acids and bases ([Houseley and Tollervey, 2009](#b0145), [Pogocki and Schöneich, 2000](#b0335)). In two publications on mRNA hydrolysis the authors state that the base sequence and secondary structure of mRNA influence the rate of hydrolysis ([Kaukinen et al., 2002](#b0200), [Mikkola et al., 2001](#b0275)). Specifically, base-stacking may decrease the cleavage rate of phosphodiester bonds. The ‘average unpaired probability’ of an mRNA molecule can be minimized. Specially designed algorithms that select nucleotide sequences for single stranded mRNA with maximum double stranded regions are available. In vitro stability is claimed to be significantly improved following this approach ([Wayment-Steele et al., 2020](#b0455)). A difference between the CureVac, Pfizer/BioNTech and Moderna vaccines is that the latter two have single nucleoside incorporations of 1-methyl-pseudouridine. A previous study has shown that this modification improves RNA secondary structure stability ([Mauger et al., 2019](#b0265)). CureVac uses GC-enrichment, which should have a similar effect ([Hubert, 2021](#b0150), [Zhang et al., 2011](#b0475)).

![](https://proxy-prod.omnivore-image-cache.app/0x187,sgbCXl-JZXTdyDiauUJvv4wu_IDj8Slgni9x3ODAbr4U/https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr5.jpg)
1. [Download : Download high-res image (114KB)](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr5%5Flrg.jpg "Download high-res image (114KB)")
2. [Download : Download full-size image](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr5.jpg "Download full-size image")

Fig. 5. Base-catalyzed intramolecular hydrolysis of the [phosphodiester bond](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/phosphodiester-bond "Learn more about phosphodiester bond from ScienceDirect's AI-generated Topic Pages") in RNA by way of a 2′,3′-cyclic phosphate. B denotes a Brønsted base. Redrawn from [Pogocki and Schöneich (2000](#b0335)).

Oxidation, in contrast, affects the nucleobases and to a lesser extent the sugar groups of the mRNA's ribose units. Oxidation can lead to the cleavage of bases, strand break and the alteration of the secondary structure of the mRNA ([Pogocki and Schöneich, 2000](#b0335)). However, as stated before, hydrolysis appears to be accepted as the main factor driving mRNA degradation ([Fabre et al., 2014](#b0120)).

### 3.2\. Analyzing mRNA stability

As the integrity and purity of the mRNA are essential to safeguard efficacy and safety of mRNA vaccines, it is important to have tools to monitor these aspects. There are various attributes of the mRNA molecules that collectively determine whether the product can be used. The review by Poveda _et al._ highlights these attributes and briefly touches upon approaches that are used in general to measure and monitor them ([Poveda et al., 2019](#b0345)).

[Table 3](#t0015) provides a comprehensive list of analytical methods to determine and monitor quality attributes and stability of mRNA vaccine bulk drug substance and final drug product. Quality specifications for the presently accepted mRNA-LNP COVID-19 products are still not available in the public domain. Leaked documents (through a cyber-attack of the EMA, just before the approval date in the UK and the USA), which contain classified information on the quality of Comirnaty, mention percentages of mRNA integrity between 70 and 75% ([Tinari, 2021](#b0420)) and the assessment report from the EMA reads: ’the quality of this medicinal product, submitted in the emergency context of the current (COVID-19) pandemic, is considered to be sufficiently consistent and acceptable.’

Table 3. Assays to determine and monitor mRNA drug substance and mRNA-LNP drug product quality attributes and stability.[1)](#tblfn3)

| **Assay**[2)](#tblfn4)                                                             | **Attribute**                                                                                             |
| ---------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
| **Characterizing DNA templates and RNA transcripts**                               |                                                                                                           |
| DNA template sequencing/mRNA sequencing                                            | Identification of mRNA                                                                                    |
| UV spectroscopy (A260 nm, A260/A280, A260/A230)                                    | Quantification - purity dependent                                                                         |
| Fluorescence-based assays (e.g., residual DNA)                                     | Quantification – purity dependent                                                                         |
| Agarose/acrylamide electrophoresis                                                 | Molecular mass, RNA integrity and quantification                                                          |
| Reverse transcriptase qPCR                                                         | Identification and quantification of mRNA                                                                 |
| Western blot for dsRNA                                                             | Quality assessment                                                                                        |
| mRNA capping analysis                                                              | Quality assessment                                                                                        |
| mRNA polyadenylated tail analysis                                                  | Quality assessment                                                                                        |
| Chromatographic assays: RP-HPLC, SE-HPLC, IP-HPLC and IEX-HPLC                     | Quantity and quality assessment                                                                           |
| **Characterizing mRNA-encoded translation products**                               |                                                                                                           |
| In vitro translation - cell free medium                                            | Translation into target protein                                                                           |
| Messenger RNA evaluation using various cell-based systems                          | Translation product analysis and potential toxicity assay                                                 |
| **Characterizing mRNA-lipid complexes**                                            |                                                                                                           |
| DLS                                                                                | Particle size (distribution)                                                                              |
| Laser Doppler electrophoresis                                                      | Zeta potential                                                                                            |
| NTA/TRPS                                                                           | Particle size (distribution)                                                                              |
| SE-HPLC(-MALS)                                                                     | Particle size distribution; assessing bound/unbound mRNA                                                  |
| Microscopy (cryo TEM, ESEM, AFM)                                                   | Nanoparticle morphology, particle size (distribution)                                                     |
| Gel or capillary electrophoresis                                                   | Assessing bound/unbound mRNA and surface charge                                                           |
| Chromatographic assays: RP-HPLC, SE-HPLC, IP-HPLC and IEX-HPLC / mass spectrometry | Quantification and integrity of lipids and/or mRNA; for some: assessing bound/unbound mRNA and molar mass |
| Fluorescent dyes                                                                   | Encapsulation efficiency                                                                                  |
| **General pharmaceutical tests**                                                   |                                                                                                           |
| Appearance, pH, osmolality, endotoxin concentration, sterility                     |                                                                                                           |

1)

Adapted from [Poveda et al., 2019](#b0345), [Muralidhara et al., 2016](#b0285); [Fan et al., 2021](#b0125); [Crommelin et al., 2021](#b0060).

2)

Abbreviations: AF4, asymmetrical flow field-flow fractionation; AFM, [atomic force microscopy](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/atomic-force-microscopy "Learn more about atomic force microscopy from ScienceDirect's AI-generated Topic Pages"); dsDNA, double stranded DNA; DLS, [dynamic light scattering](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/dynamic-light-scattering "Learn more about dynamic light scattering from ScienceDirect's AI-generated Topic Pages"); ESEM, environmental scanning electron microscopy; IEX-HPLC, ion-exchange high performance liquid chromatography; IP-HPLC, ion-pair high performance liquid chromatography; MALS, multi-angle light scattering; [NTA](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/nitrilotriacetic-acid "Learn more about NTA from ScienceDirect's AI-generated Topic Pages"), nanoparticle tracking analysis; qPCR, quantitative polymerase chain reaction; RP-HPLC, reversed-phase high performance liquid chromatography; SE-HPLC, size-exclusion high performance liquid chromatography; TEM, [transmission electron microscopy](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/transmission-electron-microscopy "Learn more about transmission electron microscopy from ScienceDirect's AI-generated Topic Pages"); TRPS, tunable resistive pulse sensing.

Here we elaborate on the techniques that are applicable for studying the integrity of mRNA and used for quality control (QC). An overview of quality documentation required by EU authorities for an Investigational Medicinal Products Dossier (IMPD) with a focus on mRNA vaccines is given by Schmidt and the interested reader is referred to that informative document for details ([Schmid, 2017](#b0380)). Because many of the described methods are intended for assaying naked mRNA, whereas mRNA vaccines for COVID-19 are encapsulated in LNPs, for some assays the mRNA first needs to be released and isolated from the LNP. Potential effects of this isolation process can be taken into account by including proper controls.

Gel [electrophoresis](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/electrophoresis "Learn more about electrophoresis from ScienceDirect's AI-generated Topic Pages") is a frequently used technique that can give information on the size and the integrity of the mRNA. When mRNA degrades and strand breaks occur, the mRNA strand becomes shorter. As a result, the intensity of the band at the expected mRNA length will decrease or the band will broaden, while new bands may appear. Using this approach Démoulins _et al._ analyzed the quality of SAM under [RNase](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/ribonuclease "Learn more about RNase from ScienceDirect's AI-generated Topic Pages") free conditions ([Démoulins et al., 2017](#b0070)). Based on the gel electrophoresis data, they were able to determine whether the mRNA is of acceptable quality and stable. Commercial electrophoresis equipment is now available for high-throughput analysis of mRNA ([Pocernich et al., 2019](#b0330)).

Recently, Zhang _et al._ showed that fluorescence correlation spectroscopy (FCS) can be used to monitor changes in mRNA size ([H. Zhang et al., 2020](#b0480)). The Brownian motion of fluorescent mRNA gives an indication of its molecular weight. This technique, however, needs a [fluorescent label](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/fluorescent-dye "Learn more about fluorescent label from ScienceDirect's AI-generated Topic Pages"), is not more accurate than gel electrophoresis and only detects mRNA degradation involving substantial changes in molecular weight, [e.g.](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/eudragit "Learn more about e.g. from ScienceDirect's AI-generated Topic Pages"), strand breaks.

Throughout the (bio)pharmaceutical industry chromatographic techniques form a powerful platform to monitor purity and stability of active pharmaceutical ingredients and drug products. However, the development of HPLC techniques for establishing purity and stability of mRNA molecules has been slow so far. Successful protocols for analyzing large mRNA molecules can be found in the patent literature as well as in publications where RP-HPLC, SE-HPLC, IP-HPLC and IEX-HPLC methods are described ([Issa and Packer, 2019](#b0155), [Kanavarioti, 2019](#b0185), [Spivak et al., 2014](#b0395)). Some of these techniques can also be used for mRNA purification purposes ([Kim et al., 2007](#b0205), [Levine et al., 2019](#b0245)). IEX-HPLC can be used for measuring free and LNP encapsulated mRNA. An orthogonal technique to establish the same parameter is the earlier mentioned RiboGreen technique. Different outcomes, however, were found for the same mRNA-LNP product; the IEX-HPLC outcome correlated better with the _in vivo_ readout than the RiboGreen data. This illustrates that even established techniques like the RiboGReen assay should be carefully validated on a case-by-case basis ([Schariter et al., 2019](#b0375)).

mRNA degradation can also be analyzed by using the reverse transcription quantitative polymerase chain reaction (RT-qPCR). This approach quantifies the total amount of mRNA that can be transcribed into intact cDNA targets for PCR. This means that all degradation that prevents this can be quantified indirectly. In their evaluation of this technique, Brisco and Morley argue that it gives reliable quantitative results ([Brisco and Morley, 2012](#b0045)). Another advantage of this method is that all types of degradation that affect transcription are taken into account, whereas the previously discussed techniques only look at mRNA size. However, RT-qPCR sometimes is less reliable due to the error rate of the enzymes that are used ([Schmid, 2017](#b0380)). Other disadvantages of this approach are that it is not widely used and the different types of degradation cannot be distinguished from each other.

mRNA expression _in vitro_ (in cells) is also used to analyze the integrity of mRNA ([Zhang et al., 2020b](#b0480); [Zhao et al., 2020](#b0490)). By using mRNA that encodes a fluorescent protein, the mRNA integrity can indirectly be determined by measuring the fluorescence signal. This approach can assist in providing guiding principles for bioactive mRNA-LNP design and formulation development studies. Alternatively, the translation efficacy of mRNA encoding non-fluorescent antigens can be determined using an enzyme-linked immunosorbent assay (ELISA), or [western blotting techniques](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/western-blot "Learn more about western blotting techniques from ScienceDirect's AI-generated Topic Pages") as has been used previously to analyze the stability of mRNA encoding the receptor-binding domain (RBD) of SARS-CoV-2 ([Zhang et al., 2020b](#b0480)). Advantages of this approach are that it gives the sum of the overall integrity of the formulation and everything that can influence the transcription of the mRNA. Disadvantages of this method are that it is not very precise, unable to show the type of mRNA damage, and time-consuming.

### 3.3\. LNP stability

Besides mRNA integrity, stability of LNPs is critical for the quality of mRNA-LNP vaccines. No information on LNP stability tests could be found for the present mRNA COVID-19 vaccines, but the siRNA-LNP formulation of Onpattro (patisaran) has a three-year shelf life when kept between 2° and 8 °C ([EMA, 2018](#b0095)). There is a warning in this SMPC text that the dispersion should not be frozen. The composition of the Onpattro LNP system is: DLin-MC3-DMA, as ionizable cationic lipid: [DSPC](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/phosphorylcholine "Learn more about DSPC from ScienceDirect's AI-generated Topic Pages"): cholesterol: PEG2000-C-DMG (see [table 1](#t0005)) (mol-ratio 50:10:38.5:1.5 mol%); they are composition-wise similar to the LNPs in Comirnaty and mRNA-1273\. A study on siRNA-LNPs composed of the lipidoid 306O13 instead of DLin-MC3-DMA, showed that in aqueous conditions the formulation remained stable at 2 °C for 156 days at pH 7 ([Ball et al., 2016](#b0025)). Particle size and siRNA encapsulation for this formulation did not significantly change. Complementary research by Suzuki et al. showed that siRNA-LNPs are stable over the experimental period, i.e. 1.5 years, at 4 °C ([Suzuki et al., 2015](#b0405)). Altogether, these data strongly indicate that instability of the mRNA, rather than LNP instability, determines the storage conditions and shelf life of the current mRNA-LNP COVID-19 vaccines.

The stability and quality attributes of [liposomes](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/liposome "Learn more about liposomes from ScienceDirect's AI-generated Topic Pages") and LNPs have been reviewed by Fan _et al._ ([Fan et al., 2021](#b0125)). LNPs can undergo chemical and physical instability. Chemical instability comprises the degradation of the lipids in the LNPs that are susceptible to hydrolysis and oxidation. Lipid oxidation can occur in [unsaturated fatty acid](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/unsaturated-fatty-acid "Learn more about unsaturated fatty acid from ScienceDirect's AI-generated Topic Pages") moieties (not present in Comirnaty and mRNA-1273) and with cholesterol, potentially as a result of a [hydroperoxide](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/hydroperoxide "Learn more about hydroperoxide from ScienceDirect's AI-generated Topic Pages") attack, an impurity present in the PEG-group of PEG2000-C-DMG ([Jaeger et al., 1994](#b0165), [Wang et al., 2019](#b0445)). Oxidative impurities may also result in oxidation of encapsulated mRNA. The carboxylic ester bonds in lipids, such as DSPC and the ionizable cationic lipids, are susceptible to temperature- and pH-dependent hydrolysis ([Fig. 6](#f0030)).

![](https://proxy-prod.omnivore-image-cache.app/0x237,sVNOk-OfI5ZruhNqDreLO-jBNPBlt60q6Qo62Te9QhBM/https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr6.jpg)
1. [Download : Download high-res image (215KB)](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr6%5Flrg.jpg "Download high-res image (215KB)")
2. [Download : Download full-size image](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr6.jpg "Download full-size image")

Fig. 6. Lipids used in the mRNA-LNP COVID-19 vaccines BNT162b2 (Comirnaty) and mRNA-1273.

Another key aspect of LNP stability is physical degradation. There are three main types of physical instability that can occur: aggregation, fusion, and leakage of the encapsulated pharmaceutical ingredient. Aggregation of LNPs during storage and fusion of LNPs has been reported ([Ayat et al., 2019](#b0015), [Ball et al., 2016](#b0025)). To increase stability on the shelf, LNPs are often formulated with PEG-lipids ([Burke et al., 2013](#b0050), [Ryals et al., 2020](#b0365)). The PEG-molecules at the surface prevent the individual LNPs from aggregating. The other type of physical degradation –leakage of the mRNA– mainly affects the stability of the encapsulated product. Of note, encapsulation efficiencies are typically > 90% and release of the RNA payload from LNPs during storage has not been reported (as measured with the RiboGreen test). mRNA that is not encapsulated (‘naked mRNA’) is hardly taken up by cells; besides, it degrades rapidly and is, therefore, not available for translation.

[Hypersensitivity reactions](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/allergic-reaction "Learn more about Hypersensitivity reactions from ScienceDirect's AI-generated Topic Pages") –rarely observed upon [intramuscular injection](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/intramuscular-injection "Learn more about intramuscular injection from ScienceDirect's AI-generated Topic Pages") of the mRNA-LNP COVID-19 vaccines– may be related to the PEG-lipids. Therefore, alternative lipids to prevent aggregate formation have been investigated. Introduction of polysarcosine-modified lipids stabilized lipid-based systems against aggregation while reducing the immunostimulatory response ([Nogueira et al., 2020](#b0290)). Additional testing is needed to establish whether such PEG-lipid alternatives improve the mRNA stability (e.g., owing to lack of peroxides).

### 3.4\. Analyzing LNP stability

Analytical methods to monitor the stability of LNPs have been expertly reviewed in the previously mentioned article by Fan et al. and Kim et al. and we refer the interested reader to this literature source ([Fan et al., 2021](#b0125); [Kim et al., 2021](#b0210)).

### 3.5\. Which mRNA-LNP component is more unstable: The mRNA, the lipids, or the combination?

To date, several studies have investigated ways to stabilize mRNA and to stabilize LNPs during storage ([Ball et al., 2016](#b0025), [Jones et al., 2007](#b0175)). It is, however, interesting to question which of these components is the bottleneck for stability. Is it the mRNA that degrades when the mRNA-LNP formulations are not frozen, are the LNPs causing the problem, or is it the combination of mRNA with the LNPs?

LNP systems entrapping chemically modified and highly stable siRNA molecules (e.g. Onpattro) have significantly longer shelf lives as compared to mRNA-LNP systems. This would suggest that not the LNP, but rather the mRNA is the current stability bottleneck.

To date, there is no research report available in the public domain on the integrity of both mRNA and LNPs in mRNA-LNP formulations. In the few studies in which the effect of storage is investigated, such as the research by Zhang _et al._ ([N.-N. Zhang et al., 2020](#b0470)), long-term effects are not measured. Therefore, we will be looking first at the long-term stability of naked mRNA, with the caveat that this may be different from the stability of mRNA encapsulated in LNPs (see below).

In their review Pascolo _et al._ remark that aqueous solutions of naked mRNA can be stored at 4 °C for only a few days, provided that the mRNA is protected from degradation by contaminating ribonucleases by an [RNase inhibitor](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/ribonuclease-inhibitor "Learn more about RNase inhibitor from ScienceDirect's AI-generated Topic Pages") ([Pascolo, 2008](#b0305)). This seems to be in line with the current view of mRNA instability. The existing body of research on the long-term stability of naked mRNA suggests that mRNA needs to be frozen or dried in order to stay stable for longer periods of time.

In 2009, Roesler _et al._ show that the translation efficacy of mRNA encoding [luciferase](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/luciferase "Learn more about luciferase from ScienceDirect's AI-generated Topic Pages") begins to decrease after 8 days or 16 days when stored –RNAse free– at room temperature as a liquid or in a lyophilized form, respectively. No efforts were made to optimize the formulations in terms of excipients and freeze drying conditions ([Fig. 7](#f0035)) ([Roesler et al., 2009](#b0360)). Conclusions about the long-term stability at refrigerator temperatures can not been drawn from this study, as the stability was only followed for 32 days. There is also information based on theoretical cleavage rate calculations. Wayment-Steele _et al._ predict that mRNA (4000 nt) will have a half-life of 941 days when it is stored at pH 7.4 and 5 °C ([Wayment-Steele et al., 2020](#b0455)). They do note that longer mRNA sequences, such as SAMs, are more prone to hydrolysis. Because this is a theoretical calculation that is only based on hydrolytic degradation kinetics, it might underestimate mRNA degradation, e.g., when trace amounts of RNase are present. More research into the stability of mRNA would have to be done to find a definitive answer.

![](https://proxy-prod.omnivore-image-cache.app/0x234,s4dZEexfvoyk2F6ADFBsbee0T-rs0ina1ZzmaCN7vE8Q/https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr7.jpg)
1. [Download : Download high-res image (211KB)](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr7%5Flrg.jpg "Download high-res image (211KB)")
2. [Download : Download full-size image](https://ars.els-cdn.com/content/image/1-s2.0-S0378517321003914-gr7.jpg "Download full-size image")

Fig. 7. Stability of mRNA in water analysed by [luciferase](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/luciferase "Learn more about luciferase from ScienceDirect's AI-generated Topic Pages") expression in transfected BHK-21 cells. Courtesy of [Roesler et al. (2009](#b0360)).

Together, these studies indicate that in an aqueous solution mRNA likely is less stable than LNPs. However, it should be reiterated that results on these two components in isolation could differ from the situation of mRNA encapsulated by LNPs. As discussed in previous sections, mRNA is located inside the LNP core together with an ionizable cationic lipid, cholesterol and water (cf. [Fig. 4](#f0020)) ([Arteta et al., 2018](#b0010)). This would mean that the mRNA is in an aqueous environment and therefore is susceptible to hydrolysis. The mechanisms of hydrolysis might then be similar to those for mRNA in solution ([Brader et al., 2021](#b0040)). However, on the other hand, mRNA inside the LNPs may be coated by the ionizable lipid through hydrophobic, hydrogen-bond and/or electrostatic interactions. In that case the mRNA might be more stable than naked mRNA dissolved in an aqueous medium. Without further studies, it can only be concluded that mRNA instability dictates storage conditions and drives the search for improved, more stable formulations.

### 3.6\. Reasons for different storage conditions

Another interesting aspect of the current mRNA vaccines is that the reported storage temperatures and corresponding ‘shelf lives’ vary widely: from −80 °C to 2–8 °C and from days to months ([Crommelin et al., 2021](#b0060)). Is it possible to identify a difference in the formulation of the mRNA vaccines that could cause this variation? Or, could the difference in storage conditions be related to the limitations of the [thermostability](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/thermostability "Learn more about thermostability from ScienceDirect's AI-generated Topic Pages") testing protocol or a more guarded approach ([Schmid, 2017](#b0380))? This information is important, as insights into the factors that positively affect the stability may be an interesting starting point for the future design of thermostable mRNA vaccines.

Tom Madden, the CEO of Acuitas Therapeutics, stated that the Moderna and Pfizer/BioNTech mRNA vaccines likely have the same stability ([Dolgin, 2020](#b0075)). Is it possible that the latter use a more conservative approach to ensure stability? Still, it is quite likely that the Pfizer/BioNTech mRNA-LNP vaccine, that currently has to be stored between −60 to −80 °C was also tested at higher temperatures and under refrigerated conditions as done by CureVac scientists ([Stitz et al., 2017](#b0400)). Moreover, the analysis techniques in the stability tests could differ in sensitivity as could the acceptance criteria. Publication of the stability study reports would provide the answers and it would be interesting to run comparative studies.

## 4\. Opportunities for improving shelf life & storage conditions

When freezing is needed to keep a vaccine stable, distribution, storage and handling by the end-user are highly protocolized and costs increase. For distribution purposes, vaccine stability at refrigerator temperatures, 2–8 °C, would be a desired improvement. In the following section we will focus on research to achieve such storage conditions, as these are seen as a considerable inconvenience for large scale use of mRNA vaccines, such as in the COVID-19 pandemic.

### 4.1\. Excipients

The conclusion from the previous section on the stability of mRNA vaccines is that in order to create more stable mRNA-LNP vaccines, stabilizing the mRNA is the first target. Obviously, any excipient used for mRNA-based vaccines must be RNase free. A review by Muralidhara _et al._ captures a lot of information on the impact of excipients and formulation milieu ([Muralidhara et al., 2016](#b0285)).

Some of the excipients proposed by Muralidhara _et al._ are already used in the mRNA vaccines by Moderna and Pfizer/BioNTech. Excipients in the formulations serve as buffers, osmolytes and [cryoprotectant](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/cryoprotective-agent "Learn more about cryoprotectant from ScienceDirect's AI-generated Topic Pages") or have a dual effect. Moderna, for example, uses a Tris–HCl buffer that would have an additional stabilizing effect on nucleic acid macromolecules as it is also a [hydroxyl radical](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/hydroxyl-radical "Learn more about hydroxyl radical from ScienceDirect's AI-generated Topic Pages") scavenger ([EMA, 2021](#b0080)). When selecting these excipients, one should keep in mind that the product may need to be stored at sub-zero temperatures and excipients affect that milieu. The choice of the buffering system and osmolyte is important as the pH may change upon freezing, as has been shown for sodium phosphate buffered systems; a 3.5 pH-units drop occurs upon freezing. [Histidine](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/histidine "Learn more about Histidine from ScienceDirect's AI-generated Topic Pages") buffers are more ‘pH-resistant’ upon freezing. But still, the pH may drop 0.5 unit when cooling from 0 °C to −30 °C ([Kolhe et al., 2010](#b0220)). And, another example, NaCl (osmolyte)-solutions have a eutectic temperature of −21 °C. Other excipients that could be added are antioxidants, non-reducing free radical scavengers (e.g., ethanol) or metal [chelators](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/chelating-agent "Learn more about chelators from ScienceDirect's AI-generated Topic Pages") ([Evans et al., 2000](#b0105)). However, the question remains to what extent they indeed ameliorate the stability of mRNA-LNP formulations during storage below or above 0 °C.

pH optimization is also important for mRNA vaccine stability, as the pH influences the hydrolysis rate of mRNA and also LNP stability. Generally, mRNA is most stable in a weakly basic environment. The pH of the Moderna and Pfizer/BioNTech vaccines is between 7 and 8\. Wayment-Steele et al. make the point that the apparent pH at the surface of the cationic, fully charged lipids could be higher than in the immediate surrounding aqueous medium ([Wayment-Steele et al., 2020](#b0455)).

### 4.2\. Lyophilization

As the presence of water initiates degradation reactions in mRNA-LNPs, lyophilization would be a logical step to improve the long-term stability of mRNA-LNP formulations. The head of viral vaccines research at Pfizer, Philip Dormitzer, has already mentioned Pfizer’s aspiration to use lyophilization for mRNA-LNP SARS-CoV-2 vaccines ([Dolgin, 2020](#b0075)). Moreover, a lyophilized form of a mRNA-based [cytomegalovirus vaccine](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/cytomegalovirus-vaccine "Learn more about cytomegalovirus vaccine from ScienceDirect's AI-generated Topic Pages") (mRNA-1647) is used in a phase 2 clinical trial. It has a claimed shelf life at 5 °C of ≥ 18 months. However, no details on the formulation and production process can be found in the public domain ([Moderna, 2021](#b0280)).

Freeze drying is widely used for live, attenuated virus vaccines ([Hansen et al., 2015](#b0135)). It has also already been investigated for naked mRNA formulations, demonstrating its applicability and beneficial effect on mRNA stabilisation. Previous research by Jones _et al._ shows that freeze-dried mRNA formulated with [trehalose](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/trehalose "Learn more about trehalose from ScienceDirect's AI-generated Topic Pages") is stable at 4 °C for up to 10 months ([Jones et al., 2007](#b0175)). Lipid [nanoparticles](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/nanoparticle "Learn more about nanoparticles from ScienceDirect's AI-generated Topic Pages") can be successfully freeze dried as well. During the freeze-drying process the structures are exposed to stress. Therefore, lyoprotectants that stabilize these [colloidal particles](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/colloidal-particle "Learn more about colloidal particles from ScienceDirect's AI-generated Topic Pages") should be included in the formulation. Sucrose or trehalose are used for that purpose ([Abdelwahed et al., 2006](#b0005)). Thus, the studies with either mRNA or with LNPs suggest that lyophilization could be a possible way to increase the stability of the combination, mRNA-LNP, and could thereby allow for storage at higher temperatures than those currently required. However, lyophilization does have its downsides, as it requires reconstitution before administration and is a relatively expensive, energy- and time-consuming process. On the other hand, keeping the mRNA vaccines (deep)frozen also comes at a cost. Therefore, a logical next step is to investigate whether lyophilization is a viable option for mRNA-LNP formulations.

Shirane _et al._ freeze dried the ethanol-containing siRNA-LNP dispersion obtained immediately after LNP formation and found no difference between the gene knockdown efficiency of the freshly prepared (ethanol removed by ultrafiltration) and the reconstituted freeze-dried formulations _in vivo_ ([Shirane et al., 2018](#b0390)). These results show promise for the feasibility of lyophilization of mRNA-LNPs, while recognizing the structural differences between siRNA and mRNA.

There is little published data on the lyophilization of mRNA-LNPs, but there is one recent paper by Zhao _et al._ studying mRNA lipid-like nanoparticles ([Zhao et al., 2020](#b0490)). They studied the effect of freeze-drying of these nanoparticles that contain an ionizable cationic lipid-like molecule. No details of the -freeze-drying process are provided. Contrary to the data on siRNA-LNPs obtained by Shirane et al., they found that after lyophilization and reconstitution the _in vitro_ test showed no loss of activity, but the _in vivo_ efficiency was lost. Thus, freeze-drying of mRNA-LNPs might be a more difficult task than previous research suggests. It would be helpful to elucidate the mechanism behind this nullification of _in vivo_ efficiency. It could, for example, be that the formulation was suboptimal for freeze-drying or that the lyophilization process itself was flawed. The authors speculate that an alteration in the [nanostructure](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/nanomaterial "Learn more about nanostructure from ScienceDirect's AI-generated Topic Pages") of the mRNA-LLPs may have caused the low _in vivo_ efficacy, as such a change could affect properties like the binding to [serum proteins](https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/plasma-protein "Learn more about serum proteins from ScienceDirect's AI-generated Topic Pages"), which are absent in the _in vitro_ study. Therefore, further improvement of the formulation excipients and freeze-drying conditions might lead to successful lyophilization of mRNA-LLPs.

Another approach, in case lyophilization of the fully prepared mRNA-LNPs complex is problematic, is to lyophilize naked mRNA and combine it with the LNPs shortly before administration. Ball _et al._ successfully followed the opposite approach by reconstituting freeze dried LNPs with siRNA/ethanol solutions ([Ball et al., 2016](#b0025)). They correctly pointed out that: ‘Unfortunately, the addition of ethanol to reconstitution solutions is often neither convenient nor practical, as dialysis into aqueous buffer would be required before use in animals or in the clinic’ If one wishes to avoid freeze drying or organic solvents for the generation of active mRNA-LNPs, one may mix an ’empty’ LNP aqueous suspension with ’fresh’ mRNA and find that the mRNA is taken up and active ([Leavitt et al., 2020](#b0235)).

Apart from challenges to preserve the integrity of mRNA-LNP through lyophilization, there are other hurdles such as the high energy consumption of the process and other costs, e.g., those related to the necessary, dramatic expansion of the lyophilization capacity worldwide in times of a pandemic. Therefore, it is worthwhile to consider alternative drying techniques. Only one publication could be found on spray drying of mRNA-LNPs where polymers (e.g., Eudragit) were needed as stabilizers for mRNA-LNPs to secure translation efficiency _in vivo_ ([Karve et al., 2020](#b0190)). Supercritical drying techniques would be another alternative for freeze-drying; they have been shown to be feasible –with their pros and cons– for other macromolecular biotech products, such as proteins ([Jovanović et al., 2004](#b0180)).

## 5\. Conclusions and prospects

This review outlines how different aspects of the current mRNA vaccine formulations influence their stability during storage. We conclude that exposure of mRNA to water likely is the main factor for mRNA vaccine instability. An implication of this is that decreasing the exposure to water would be a promising approach for improving mRNA vaccine stability.

The studies on mRNA-LNP structures indicate that the mRNA is located in the core of LNPs together with ionizable cationic lipid and water. This raises important questions about the possible shielding of the mRNA from water. It is, for example, unknown if and how the ionizable cationic lipids in the LNP interact with the mRNA. More work needs to be done to confirm the proposed structure and to understand the consequences. For instance, the pH inside the LNPs has been identified as an important characteristic to study in relation to stability. Another object of study would be to specifically analyze the type(s) of degradation that mRNA molecules undergo in their final formulation and whether sequence adjustment could help to maintain strand integrity. This could then also be coupled to characterization and optimization of the secondary and tertiary structure of mRNA, as there are indications that some folded structures are more stable.

This report is the first comprehensive survey of the factors behind mRNA-LNP vaccine instability. It also points towards solutions to address this instability and thereby may be of assistance to the development of more thermostable mRNA-LNP vaccines, alleviating a major barrier for the distribution of these vaccines.

## CRediT authorship contribution statement

**Linde Schoenmaker:** Conceptualization, Writing - review & editing. **Dominik Witzigmann:** Writing - review & editing. **Jayesh A. Kulkarni:** Writing - review & editing. **Rein Verbeke:** Writing - review & editing. **Gideon Kersten:** Writing - review & editing. **Wim Jiskoot:** Conceptualization, Writing - review & editing. **Daan Crommelin:** Conceptualization, Writing - review & editing.

## Declaration of Competing Interest

The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.

## Acknowledgement

We acknowledge Mr. Oscar Escalona Rayo, Leiden University, for expertly redrawing [Fig. 5](#f0025).

## References

1. [ABDELWAHED et al., 2006](#bb0005)  
Freeze-drying of nanoparticles: Formulation, process and storage considerations
2. [Yanez Arteta et al., 2018](#bb0010)  
M. Yanez Arteta, T. Kjellman, S. Bartesaghi, S. Wallin, X. Wu, A.J. Kvist, A. Dabkowska, N. Székely, A. Radulescu, J. Bergenholtz, L. Lindfors  
Successful reprogramming of cellular protein production through mRNA delivered by functionalized lipid nanoparticles
3. [Ayat et al., 2019](#bb0015)  
N.R. Ayat, Z. Sun, D.a. Sun, M. Yin, R.C. Hall, A.M. Vaidya, X. Liu, A.L. Schilb, J.H. Scheidt, Z.-R. Lu  
Formulation of biocompatible targeted ECO/siRNA nanoparticles with long-term stability for clinical translation of RNAi
4. [Baden et al., 2021](#bb0020)  
L.R. Baden, H.M. El Sahly, B. Essink, K. Kotloff, S. Frey, R. Novak, D. Diemert, S.A. Spector, N. Rouphael, C.B. Creech, J. McGettigan, S. Khetan, N. Segall, J. Solis, A. Brosz, C. Fierro, H. Schwartz, K. Neuzil, L. Corey, P. Gilbert, H. Janes, D. Follmann, M. Marovich, J. Mascola, L. Polakowski, J. Ledgerwood, B.S. Graham, H. Bennett, R. Pajon, C. Knightly, B. Leav, W. Deng, H. Zhou, S. Han, M. Ivarsson, J. Miller, T. Zaks  
Efficacy and safety of the mRNA-1273 SARS-CoV-2 vaccine
5. [Ball et al., 2016](#bb0025)  
R.L. Ball, P. Bajaj, K.A. Whitehead  
Achieving long-term stability of lipid nanoparticles: examining the effect of pH, temperature, and lyophilization
6. [Bloom et al., 2020](#bb0035)  
K. Bloom, F. van den Berg, P. Arbuthnot  
Self-amplifying RNA vaccines for infectious diseases
7. [Brader et al., 2021](#bb0040)  
M.L. Brader, S.J. Williams, J.M. Banks, W.H. Hui, Z.H. Zhou, L. Jin  
Encapsulation state of messenger RNA inside lipid nanoparticles
8. [Brisco and Morley, 2012](#bb0045)  
M.J. Brisco, A.A. Morley  
Quantification of RNA integrity and its use for measurement of transcript number
9. [Burke et al., 2013](#bb0050)  
Burke, P.A., Gindy, M.E., Mathre, D.J., Kumar, V., Prud’homme, R.K., 2013\. Preparation of Lipid Nanoparticles. US 2013/0037977.
10. [Buschmann et al., 2021](#bb0055)  
M.D. Buschmann, M.J. Carrasco, S. Alishetty, M. Paige, M.G. Alameh, D. Weissman  
Nanomaterial delivery systems for mRNA vaccines
11. [Crommelin et al., 2021](#bb0060)  
D.J.A. Crommelin, T.J. Anchordoquy, D.B. Volkin, W. Jiskoot, E. Mastrobattista  
Addressing the cold reality of mRNA vaccine stability
12. [CureVac, 2020](#bb0065)  
CureVac, 2020\. CureVac’s COVID-19 Vaccine Candidate, CVnCoV, Suitable for Standard Fridge Temperature Logistics \[WWW Document\]. URL https://www.curevac.com/en/2020/11/12/curevacs-covid-19-vaccine-candidate-cvncov-suitable-for-standard-fridge-temperature-logistics/ (accessed 3.19.21).
13. [Démoulins et al., 2017](#bb0070)  
T. Démoulins, P.C. Englezou, P. Milona, N. Ruggli, N. Tirelli, C. Pichon, C. Sapet, T. Ebensen, C.A. Guzmán, K.C. McCullough  
Self-Replicating RNA Vaccine Delivery to Dendritic Cells BT - RNA Vaccines: Methods and Protocols
14. [Dolgin, 2020](#bb0075)  
Dolgin, E., 2020\. COVID-19 vaccines poised for launch, but impact on pandemic unclear. Nat. Biotechnol. https://doi.org/10.1038/d41587-020-00022-y.
15. [EMA, 2021](#bb0080)  
EMA  
COVID-19 Vaccine Moderna: Summary of Product Characteristics \[WWW Document\]
16. [EMA, 2020a](#bb0085)  
EMA  
Assessment report Comirnaty Common name: COVID-19 mRNA vaccine (nucleoside-modified) \[WWW Document\]
17. [EMA, 2020b](#bb0090)  
EMA  
Comirnaty: Summary of Product Characteristics \[WWW Document\]
18. [EMA, 2018](#bb0095)  
EMA  
Onpattro: Summary of Product Characteristics \[WWW Document\]
19. [Erasmus et al., 2020](#bb0100)  
Jesse H. Erasmus, Amit P. Khandhar, Megan A. O’Connor, Alexandra C. Walls, Emily A. Hemann, Patience Murapa, Jacob Archer, Shanna Leventhal, James T. Fuller, Thomas B. Lewis, Kevin E. Draves, Samantha Randall, Kathryn A. Guerriero, Malcolm S. Duthie, Darrick Carter, Steven G. Reed, David W. Hawman, Heinz Feldmann, Michael Gale, David Veesler, Peter Berglund, Deborah Heydenburg Fuller  
An Alphavirus-derived replicon RNA vaccine induces SARS-CoV-2 neutralizing antibody and T cell responses in mice and nonhuman primates
20. [Evans et al., 2000](#bb0105)  
R.K. Evans, Z. Xu, K.E. Bohannon, B. Wang, M.W. Bruner, D.B. Volkin  
Evaluation of degradation pathways for plasmid dna in pharmaceutical formulations via accelerated stability studies
21. [Evers et al., 2018](#bb0110)  
Martijn J.W. Evers, Jayesh A. Kulkarni, Roy van der Meel, Pieter R. Cullis, Pieter Vader, Raymond M. Schiffelers  
State-of-the-art design and rapid-mixing production techniques of lipid nanoparticles for nucleic acid delivery
22. [Eygeris et al., 2020](#bb0115)  
Yulia Eygeris, Siddharth Patel, Antony Jozic, Gaurav Sahay  
Deconvoluting lipid nanoparticle structure for messenger RNA delivery
23. [Fabre et al., 2014](#bb0120)  
Anne-Lise Fabre, Marthe Colotte, Aurélie Luis, Sophie Tuffet, Jacques Bonnet  
An efficient method for long-term room temperature storage of RNA
24. [Fan et al., 2021](#bb0125)  
Yuchen Fan, Maria Marioli, Kelly Zhang  
Analytical characterization of liposomes and other lipid nanoparticles for drug delivery
25. [FDA, 2017](#bb0130)  
FDA  
ONPATTRO (patisiran) Lipid Complex Injection Addendum to Drug Product Quality Review \[WWW Document\]
26. [Hansen et al., 2015](#bb0135)  
L.J.J. Hansen, R. Daoussi, C. Vervaet, J.-P. Remon, T.R.M. De Beer  
Freeze-drying of live virus vaccines: a review
27. [Hassett et al., 2019](#bb0140)  
K.J. Hassett, K.E. Benenato, E. Jacquinet, A. Lee, A. Woods, O. Yuzhakov, S. Himansu, J. Deterling, B.M. Geilich, T. Ketova, C. Mihai, A. Lynn, I. McFadyen, M.J. Moore, J.J. Senn, M.G. Stanton, Ö. Almarsson, G. Ciaramella, L.A. Brito  
Optimization of lipid nanoparticles for intramuscular administration of mRNA vaccines
28. [Houseley and Tollervey, 2009](#bb0145)  
Jonathan Houseley, David Tollervey  
The many pathways of RNA degradation
29. [Hubert, 2021](#bb0150)  
Hubert, B., 2021\. The CureVac Vaccine, and a brief tour through some of the wonders of nature \[WWW Document\]. URL https://berthub.eu/articles/posts/curevac-vaccine-and-wonders-of-biology/ (accessed 3.18.21).
30. [Issa and Packer, 2019](#bb0155)  
Issa, W., Packer, M., 2019\. METHODS FOR HPLC ANALYSIS. WO 2019/036685.
31. [Jackson et al., 2020](#bb0160)  
N.A.C. Jackson, K.E. Kester, D. Casimiro, S. Gurunathan, F. DeRosa  
The promise of mRNA vaccines: a biotech and industrial perspective
32. [Jaeger et al., 1994](#bb0165)  
J. Jaeger, K. Sorensen, S.P. Wolff  
Peroxide accumulation in detergents  
J. Biochem. Biophys. Methods, 29 (1) (1994), pp. 77-81,  
[](https://doi.org/10.1016/0165-022X%2894%2990058-2)
33. [Jayaraman et al., 2012](#bb0170)  
M. Jayaraman, S.M. Ansell, B.L. Mui, Y.K. Tam, J. Chen, X. Du, D. Butler, L. Eltepu, S. Matsuda, J.K. Narayanannair, K.G. Rajeev, I.M. Hafez, A. Akinc, M.A. Maier, M.A. Tracy, P.R. Cullis, T.D. Madden, M. Manoharan, M.J. Hope  
Maximizing the potency of siRNA lipid nanoparticles for hepatic gene silencing in vivo  
Angew. Chemie Int. Ed., 51 (2012), pp. 8529-8533,  
[](https://doi.org/10.1002/anie.201203263)
34. [Jones et al., 2007](#bb0175)  
Kathryn L. Jones, Debbie Drane, Eric J. Gowans  
Long-term storage of DNA-free RNA for use in vaccine studies
35. [Jovanović et al., 2004](#bb0180)  
Nataša Jovanović, Andréanne Bouchard, Gerard W. Hofland, Geert-Jan Witkamp, Daan J.A. Crommelin, Wim Jiskoot  
Stabilization of proteins in dry powder formulations using supercritical fluid technology
36. [Kanavarioti, 2019](#bb0185)  
A. Kanavarioti  
HPLC methods for purity evaluation of man-made single-stranded RNAs
37. [Karve et al., 2020](#bb0190)  
Karve, S., DeRosa, F., Heartlein, M., Patel, Z., Sarode, A., 2020\. DRY POWDER FORMULATIONS FOR MESSENGER RNA. US 2020/0022921.
38. [Kauffman et al., 2016](#bb0195)  
K.J. Kauffman, M.J. Webber, D.G. Anderson  
Materials for non-viral intracellular delivery of messenger RNA therapeutics
39. [Kaukinen et al., 2002](#bb0200)  
U. Kaukinen, S. Lyytikäinen, S. Mikkola, H. Lönnberg  
The reactivity of phosphodiester bonds within linear single-stranded oligoribonucleotides is strongly dependent on the base sequence
40. [Kim et al., 2007](#bb0205)  
I. Kim, S.A. Mckenna, E.V. Puglisi, J.D. Puglisi  
Rapid purification of RNAs using fast performance liquid chromatography (FPLC)
41. [Kim et al., 2021](#bb0210)  
J. Kim, Y. Eygeris, M. Gupta, G. Sahay  
Self-assembled mRNA vaccines
42. [Klauer and van Hoof, 2012](#bb0215)  
A. Alejandra Klauer, Ambro van Hoof  
Degradation of mRNAs that lack a stop codon: a decade of nonstop progress
43. [Kolhe et al., 2010](#bb0220)  
P. Kolhe, E.K. Amend, S. Singh  
Impact of freezing on pH of buffered solutions and consequences for monoclonal antibody aggregation
44. [Kulkarni et al., 2018](#bb0225)  
Jayesh A. Kulkarni, Maria M. Darjuan, Joanne E. Mercer, Sam Chen, Roy van der Meel, Jenifer L. Thewalt, Yuen Yi C. Tam, Pieter R. Cullis  
On the formation and morphology of lipid nanoparticles containing ionizable cationic lipids and siRNA
45. [Kulkarni et al., 2019](#bb0230)  
Jayesh A. Kulkarni, Dominik Witzigmann, Jerry Leung, Yuen Yi C. Tam, Pieter R. Cullis  
On the role of helper lipids in lipid nanoparticle formulations of siRNA
46. [Leavitt et al., 2020](#bb0235)  
Leavitt, B., Cullis, P., Petkau, T., Hill, A., Wagner, P., Kulkarni, J., 2020\. Compositions and systems comprising transfection-competent vesicles free of organic-solvents and detergents and methods related thereto. WO/2020/077007.
47. [Leung et al., 2015](#bb0240)  
Alex K.K. Leung, Yuen Yi C. Tam, Sam Chen, Ismail M. Hafez, Pieter R. Cullis  
Microfluidic mixing: a general method for encapsulating macromolecules in lipid nanoparticle systems
48. [Levine et al., 2019](#bb0245)  
Levine, A., Ono, T., Hirose, T., 2019\. HPLC Purification of mRNA with Reverse Phase and Size Exclusion Chromatography \[WWW Document\]. URL https://www.nacalai.co.jp/global/download/pdf/TIDES\_2019\_poster\_mrna.pdf (accessed 3.19.21).
49. [Liang et al., 2017](#bb0250)  
F. Liang, G. Lindgren, A. Lin, E.A. Thompson, S. Ols, J. Röhss, S. John, K. Hassett, O. Yuzhakov, K. Bahl, L.A. Brito, H. Salter, G. Ciaramella, K. Loré  
Efficient targeting and activation of antigen-presenting cells in vivo after modified mRNA vaccine administration in rhesus macaques
50. [Lindsay et al., 2019](#bb0255)  
Kevin E. Lindsay, Sushma M. Bhosle, Chiara Zurla, Jared Beyersdorf, Kenneth A. Rogers, Daryll Vanover, Peng Xiao, Mariluz Araínga, Lisa M. Shirreff, Bruno Pitard, Patrick Baumhof, Francois Villinger, Philip J. Santangelo  
Visualization of early events in mRNA vaccine delivery in non-human primates via PET–CT and near-infrared imaging
51. [Lipfert et al., 2014](#bb0260)  
Jan Lipfert, Sebastian Doniach, Rhiju Das, Daniel Herschlag  
Understanding nucleic acid-ion interactions  
Annu. Rev. Biochem., 83 (1) (2014), pp. 813-841,  
[](https://doi.org/10.1146/annurev-biochem-060409-092720)
52. [Mauger et al., 2019](#bb0265)  
David M. Mauger, B. Joseph Cabral, Vladimir Presnyak, Stephen V. Su, David W. Reid, Brooke Goodman, Kristian Link, Nikhil Khatwani, John Reynders, Melissa J. Moore, Iain J. McFadyen  
mRNA structure regulates protein expression through changes in functional half-life  
Proc. Natl. Acad. Sci., 116 (48) (2019), pp. 24075-24083,  
[](https://doi.org/10.1073/pnas.1908052116)
53. [McKay et al., 2020](#bb0270)  
P.F. McKay, K. Hu, A.K. Blakney, K. Samnuan, J.C. Brown, R. Penn, J. Zhou, C.R. Bouton, P. Rogers, K. Polra, P.J.C. Lin, C. Barbosa, Y.K. Tam, W.S. Barclay, R.J. Shattock  
Self-amplifying RNA SARS-CoV-2 lipid nanoparticle vaccine candidate induces high neutralizing antibody titers in mice
54. [Mikkola et al., 2001](#bb0275)  
S. Mikkola, U. Kaukinen, H. Lönnberg  
The effect of secondary structure on cleavage of the phosphodiester bonds of RNA
55. [Moderna, 2021](#bb0280)  
Moderna, 2021\. Cytomegalovirus (CMV) vaccine (mRNA-1647) \[WWW Document\]. URL https://investors.modernatx.com/static-files/693ffcac-b2fc-4f7e-91c5-0a9164e7c6dc (accessed 4.2.21).
56. [Muralidhara et al., 2016](#bb0285)  
Bilikallahalli K. Muralidhara, Rinku Baid, Steve M. Bishop, Min Huang, Wei Wang, Sandeep Nema  
Critical considerations for developing nucleic acid macromolecule based drug products
57. [Nogueira et al., 2020](#bb0290)  
Sara S. Nogueira, Anne Schlegel, Konrad Maxeiner, Benjamin Weber, Matthias Barz, Martin A. Schroer, Clement E. Blanchet, Dmitri I. Svergun, Srinivas Ramishetti, Dan Peer, Peter Langguth, Ugur Sahin, Heinrich Haas  
Polysarcosine-functionalized lipid nanoparticles for therapeutic mRNA delivery
58. [Pardi et al., 2018](#bb0295)  
Norbert Pardi, Michael J. Hogan, Frederick W. Porter, Drew Weissman  
mRNA vaccines — a new era in vaccinology
59. [Pardi et al., 2015](#bb0300)  
N. Pardi, S. Tuyishime, H. Muramatsu, K. Kariko, B.L. Mui, Y.K. Tam, T.D. Madden, M.J. Hope, D. Weissman  
Expression kinetics of nucleoside-modified mRNA delivered in lipid nanoparticles to mice by various routes
60. [Pascolo, 2008](#bb0305)  
S. Pascolo  
Vaccination with Messenger RNA (mRNA)  
S. Bauer, G. Hartmann (Eds.), Toll-Like Receptors (TLRs) and Innate Immunity, Springer, Berlin (2008), pp. 221-235,  
[10.1007/978-3-540-72167-3\_11](https://doi.org/10.1007/978-3-540-72167-3%5F11)
61. [Patel et al., 2020](#bb0310)  
S. Patel, N. Ashwanikumar, E. Robinson, Y. Xia, C. Mihai, J.P. Griffith, S. Hou, A.A. Esposito, T. Ketova, K. Welsher, J.L. Joyal, Ö. Almarsson, G. Sahay  
Naturally-occurring cholesterol analogues in lipid nanoparticles induce polymorphic shape and enhance intracellular delivery of mRNA
62. [Pepini et al., 2017](#bb0315)  
Pepini, T., Pulichino, A.-M., Carsillo, T., Carlson, A.L., Sari-Sarraf, F., Ramsauer, K., Debasitis, J.C., Maruggi, G., Otten, G.R., Geall, A.J., Yu, D., Ulmer, J.B., Iavarone, C., 2017\. Induction of an IFN-Mediated Antiviral Response by a Self-Amplifying RNA Vaccine: Implications for Vaccine Design. J. Immunol. 198, 4012 LP – 4024\. https://doi.org/10.4049/jimmunol.1601877.
63. [Petsch et al., 2012](#bb0320)  
Benjamin Petsch, Margit Schnee, Annette B Vogel, Elke Lange, Bernd Hoffmann, Daniel Voss, Thomas Schlake, Andreas Thess, Karl-Josef Kallen, Lothar Stitz, Thomas Kramps  
Protective efficacy of in vitro synthesized, specific mRNA vaccines against influenza A virus infection
64. [Pierrat and Lebeau, 2015](#bb0325)  
Philippe Pierrat, Luc Lebeau  
Characterization of titratable amphiphiles in lipid membranes by fluorescence spectroscopy
65. [Pocernich et al., 2019](#bb0330)  
Pocernich, C., Luttgeharm, K., Siembieda, S., 2019\. Assessment of Long IVT mRNA Fragments with the Agilent Fragment Analyzer System \[WWW Document\]. URL https://cdn.technologynetworks.com/ep/pdfs/assessment-of-long-ivt-mrna-fragments-with-the-agilent-fragment-analyzer-system.pdf (accessed 3.18.21).
66. [Pogocki and Schöneich, 2000](#bb0335)  
D. Pogocki, C. Schöneich  
Chemical stability of nucleic acid-derived drugs
67. [Polack et al., 2020](#bb0340)  
Fernando P. Polack, Stephen J. Thomas, Nicholas Kitchin, Judith Absalon, Alejandra Gurtman, Stephen Lockhart, John L. Perez, Gonzalo Pérez Marc, Edson D. Moreira, Cristiano Zerbini, Ruth Bailey, Kena A. Swanson, Satrajit Roychoudhury, Kenneth Koury, Ping Li, Warren V. Kalina, David Cooper, Robert W. Frenck, Laura L. Hammitt, Özlem Türeci, Haylene Nell, Axel Schaefer, Serhat Ünal, Dina B. Tresnan, Susan Mather, Philip R. Dormitzer, Uğur Şahin, Kathrin U. Jansen, William C. Gruber  
Safety and efficacy of the BNT162b2 mRNA covid-19 vaccine
68. [Poveda et al., 2019](#bb0345)  
C. Poveda, A.B. Biter, M.E. Bottazzi, U. Strych  
Establishing preferred product characterization for the evaluation of RNA vaccine antigens
69. [Reichmuth et al., 2016](#bb0350)  
Andreas M Reichmuth, Matthias A Oberli, Ana Jaklenec, Robert Langer, Daniel Blankschtein  
mRNA vaccine delivery using lipid nanoparticles
70. [Rissanou et al., 2020](#bb0355)  
Anastassia N. Rissanou, Andreas Ouranidis, Kostas Karatasos  
Complexation of single stranded RNA with an ionizable lipid: an all-atom molecular dynamics simulation study
71. [Roesler et al., 2009](#bb0360)  
Elisabeth Roesler, Richard Weiss, Esther E. Weinberger, Angelika Fruehwirth, Angelika Stoecklinger, Sven Mostböck, Fatima Ferreira, Josef Thalhamer, Sandra Scheiblhofer  
Immunize and disappear—Safety-optimized mRNA vaccination with a panel of 29 allergens
72. [Ryals et al., 2020](#bb0365)  
Renee C. Ryals, Siddharth Patel, Chris Acosta, Madison McKinney, Mark E. Pennesi, Gaurav Sahay, Alfred S. Lewin  
The effects of PEGylation on LNP based mRNA delivery to the eye
73. [Sahin et al., 2014](#bb0370)  
Ugur Sahin, Katalin Karikó, Özlem Türeci  
MRNA-based therapeutics — developing a new class of drugs
74. [Schariter et al., 2019](#bb0375)  
Schariter, J., Hassett, K., Smith, M., Almarsson, O., Brito, L., 2019\. METHODS OF MAKING LIPID NANOPARTICLES. WO/2019/046809.
75. [Schmid, 2017](#bb0380)  
A. Schmid  
Considerations for producing mRNA vaccines for clinical trials BT - RNA vaccines: methods and protocols
76. [Sebastiani et al., 2021](#bb0385)  
F. Sebastiani, M. Yanez Arteta, M. Lerche, L. Porcar, C. Lang, R.A. Bragg, C.S. Elmore, V.R. Krishnamurthy, R.A. Russell, T. Darwish, H. Pichler, S. Waldie, M. Moulin, M. Haertlein, V.T. Forsyth, L. Lindfors, M. Cárdenas  
Apolipoprotein E binding drives structural and compositional rearrangement of mRNA-containing lipid nanoparticles
77. [Shirane et al., 2018](#bb0390)  
Daiki Shirane, Hiroki Tanaka, Yuta Nakai, Hiroki Yoshioka, Hidetaka Akita  
Development of an alcohol dilution-lyophilization method for preparing lipid nanoparticles containing encapsulated siRNA
78. [Spivak et al., 2014](#bb0395)  
Spivak, V.B., Sharokh, Z., Issa, W.J., 2014\. ANALYSIS OF MRNA HETEROGENEITY AND STABILITY. WO 2014/144711.
79. [Stitz et al., 2017](#bb0400)  
Lothar Stitz, Annette Vogel, Margit Schnee, Daniel Voss, Susanne Rauch, Thorsten Mutzke, Thomas Ketterer, Thomas Kramps, Benjamin Petsch, Charles E. Rupprecht  
A thermostable messenger RNA based vaccine against rabies
80. [Suzuki et al., 2015](#bb0405)  
Y. Suzuki, K. Hyodo, Y. Tanaka, H. Ishihara  
siRNA-lipid nanoparticles with long-term storage stability facilitate potent gene-silencing in vivo
81. [Tanaka et al., 2020](#bb0410)  
Hiroki Tanaka, Tatsunari Takahashi, Manami Konishi, Nae Takata, Masaki Gomi, Daiki Shirane, Ryo Miyama, Shinya Hagiwara, Yuki Yamasaki, Yu Sakurai, Keisuke Ueda, Kenjirou Higashi, Kunikazu Moribe, Eiji Shinsho, Ruka Nishida, Kaori Fukuzawa, Etsuo Yonemochi, Koji Okuwaki, Yuji Mochizuki, Yuta Nakai, Kota Tange, Hiroki Yoshioka, Shinya Tamagawa, Hidetaka Akita  
Self-degradable lipid-like materials based on “hydrolysis accelerated by the intra-particle enrichment of reactant (HyPER)” for messenger RNA delivery
82. [Thess et al., 2015](#bb0415)  
Andreas Thess, Stefanie Grund, Barbara L Mui, Michael J Hope, Patrick Baumhof, Mariola Fotin-Mleczek, Thomas Schlake  
Sequence-engineered mRNA without chemical nucleoside modifications enables an effective protein therapy in large animals
83. [Tinari, 2021](#bb0420)  
S. Tinari  
The EMA covid-19 data leak, and what it tells us about mRNA instability
84. [Verbeke et al., 2019](#bb0425)  
Rein Verbeke, Ine Lentacker, Stefaan C. De Smedt, Heleen Dewitte  
Three decades of messenger RNA vaccine development
85. [Verbeke et al., 2021](#bb0430)  
R. Verbeke, I. Lentacker, S.C. De Smedt, H. Dewitte  
The dawn of mRNA vaccines: the COVID-19 case
86. [Viger-Gravel et al., 2018](#bb0435)  
Jasmine Viger-Gravel, Anna Schantz, Arthur C. Pinon, Aaron J. Rossini, Staffan Schantz, Lyndon Emsley  
Structure of lipid nanoparticles containing siRNA or mRNA by dynamic nuclear polarization-enhanced NMR spectroscopy
87. [Vogel et al., 2018](#bb0440)  
Annette B. Vogel, Laura Lambert, Ekaterina Kinnear, David Busse, Stephanie Erbar, Kerstin C. Reuter, Lena Wicke, Mario Perkovic, Tim Beissert, Heinrich Haas, Stephen T. Reece, Ugur Sahin, John S. Tregoning  
Self-amplifying RNA vaccines give equivalent protection against influenza to mRNA vaccines but at much lower doses
88. [Wang et al., 2019](#bb0445)  
Changguang Wang, Dumindika A. Siriwardane, Wenlei Jiang, Thilak Mudalige  
Quantitative analysis of cholesterol oxidation products and desmosterol in parenteral liposomal pharmaceutical formulations
89. [Ward and McCormack, 2021](#bb0450)  
D. Ward, S. McCormack  
Clinical trial to assess the safety of a coronavirus vaccine in healthy men and women
90. [Wayment-Steele et al., 2020](#bb0455)  
Wayment-Steele, H.K., Kim, D.S., Choe, C.A., Nicol, J.J., Wellington-Oguri, R., Sperberg, R.A.P., Huang, P.-S., Das, R., 2020\. Theoretical basis for stabilizing messenger RNA through secondary structure design. bioRxiv Prepr. Serv. Biol. 2020.08.22.262931\. https://doi.org/10.1101/2020.08.22.262931.
91. [World Health Organization, 2021](#bb0460)  
World Health Organization  
Draft landscape and tracker of COVID-19 candidate vaccines \[WWW Document\]
92. [Wrapp et al., 2020](#bb0465)  
Wrapp, D., Wang, N., Corbett, K.S., Goldsmith, J.A., Hsieh, C.-L., Abiona, O., Graham, B.S., McLellan, J.S., 2020\. Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Science (80-.). 367, 1260 LP – 1263\. https://doi.org/10.1126/science.abb2507.
93. [Zhang et al., 2020a](#bb0470)  
Heyang Zhang, Koen Rombouts, Laurens Raes, Ranhua Xiong, Stefaan C. De Smedt, Kevin Braeckmans, Katrien Remaut  
Fluorescence-based quantification of messenger RNA and plasmid DNA decay kinetics in extracellular biological fluids and cell extracts
94. [Zhang et al., 2011](#bb0475)  
J. Zhang, C.C.J. Kuo, L. Chen  
GC content around splice sites affects splicing through pre-mRNA secondary structures
95. [Zhang et al., 2020b](#bb0480)  
Na-Na Zhang, Xiao-Feng Li, Yong-Qiang Deng, Hui Zhao, Yi-Jiao Huang, Guan Yang, Wei-Jin Huang, Peng Gao, Chao Zhou, Rong-Rong Zhang, Yan Guo, Shi-Hui Sun, Hang Fan, Shu-Long Zu, Qi Chen, Qi He, Tian-Shu Cao, Xing-Yao Huang, Hong-Ying Qiu, Jian-Hui Nie, Yuhang Jiang, Hua-Yuan Yan, Qing Ye, Xia Zhong, Xia-Lin Xue, Zhen-Yu Zha, Dongsheng Zhou, Xiao Yang, You-Chun Wang, Bo Ying, Cheng-Feng Qin  
A Thermostable mRNA Vaccine against COVID-19
96. [Zhao and He, 2015](#bb0485)  
Boxuan Simen Zhao, Chuan He  
Pseudouridine in a new era of RNA modifications
97. [Zhao et al., 2020](#bb0490)  
Pengxuan Zhao, Xucheng Hou, Jingyue Yan, Shi Du, Yonger Xue, Wenqing Li, Guangya Xiang, Yizhou Dong  
Long-term storage of lipid-like nanoparticles for mRNA delivery

## Cited by (494)

---

